sandbox-web/src/main/java/com/exadel/sandbox/team5/ValidateOrderController.java

```diff
@@ -30,7 +30,7 @@ public ModelAndView validateQRCode(@PathVariable String uuid) {
         modelAndView.addObject("validationResult", responsePair.getFirst().getText());
         modelAndView.addObject("email", responseArray[0]);
         modelAndView.addObject("discountName", responseArray[1]);
-        modelAndView.addObject("causeOfError", responsePair.getSecond());
+        modelAndView.addObject("causeOfError", responseArray[0]);
```

Comment:
does URL contain error message 'It is the root error cause/some_url_part/123 ' ? or what does `response` contain?